.evergreen/run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       AUTH                    Set to enable authentication. Defaults to "noauth"
+#       SSL                     Set to enable SSL. Defaults to "nossl"
+#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
+#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
+
+
+AUTH=${AUTH:-noauth}
+SSL=${SSL:-nossl}
+MONGODB_URI=${MONGODB_URI:-}
+TESTS=${TESTS:-}
+PHP_VERSION=${PHP_VERSION:-7.2.21}
+
+# The PLATFORM environment variable is used by the matrix testing project.
+# For the time being, we can use that to install the correct extension version
+if [ "x${PLATFORM}" != "x" ]; then
+  DIR=$(dirname $0)
+  DRIVER_VERSION=1.5.5 . $DIR/install-dependencies.sh
```

Comment:
Ah, we lucked out with this being 1.5+, as we only standardized on the `configure` options in 1.5.0: https://github.com/mongodb/mongo-php-driver/commit/da72946e654f7a5afda189795a05a03726c1877f

Reply:
There's a fine line between careful planning and pure dumb luck 😅